src/cron/cleanZombieUsers.ts

```diff
@@ -22,9 +23,26 @@ const cron: Cron = {
       ])
       .andWhere({
         createdAt: LessThan(timeThreshold),
-      });
+      })
+      .returning(['id']);
+
+    const { affected, raw } = await query.execute();
+
+    if (Array.isArray(raw) && raw.length > 0) {
+      await con
```

Comment:
Isn't it a better approach to use a trigger on delete user?

Reply:
hmm actually yes